source/whats-new.txt

```diff
@@ -26,6 +26,7 @@ What's New
 Learn what's new in:
 
 * :ref:`Version 2.0 <version-2.0>`
```

Comment:
Q: Do you know why this page contains updates for 2.0, but that version isn't in the docs? Asking because this might affect where this branch gets merged

Reply:
I think that the 2.0 changes have been added to "upcoming," which is why it's showing up here. [This Slack thread](https://mongodb.slack.com/archives/C028XP912BF/p1706544900185739) has some info about that